### PR TITLE
AoU image: add Hail dependencies

### PR DESCRIPTION
**Pull request by calbach**
AoU ticket: https://precisionmedicineinitiative.atlassian.net/browse/RW-5412

- Add Hail to AoU image
- Bump GATK branch commit

Verified that Hail is functional with Dataproc, and pyspark fails to import on GCE, which is reasonable behavior for now.

CC @ahaessly @petesantos @ericsong @rtitle @als364

### CODE REVIEW

**Comment by calbach** [comment]
Please hold off on merging this for now - going to check whether I can resolve the PDF download issue as well

**Review by rtitle — APPROVED** [review]
It looks good to me, but I defer to AoU for the testing. 

Just as an observation: we used to have a single mega-image for FireCloud/Terra but moved to an image stack for better composability -- e.g. fewer conflicts between tools, smaller images if a user only cares about certain functionality, easier upgrades, etc. I can also see the benefit of an all-in-one image though, and it's interesting to see AoU moving in that direction.

**Comment by rtitle** [comment]
re enabling PDF, check the diff in image size. I recall that installing latex took an unreasonable amount of space, but that was also >1y ago.

**Comment by rtitle** [comment]
Also leaving this to be aware of: https://broadworkbench.atlassian.net/browse/IA-2221

I saw it on Qi's handoff doc, I don't know the details/context.

**Comment by calbach** [comment]
Thanks - I did notice yesterday that I could not pull the latest hail image, so that explains it.

I'm hitting a slow trickle of additional issues with the PDF fix I add more and more libraries.. various additional packages are needed in order to properly render. Going to punt on the PDF issue for now. So far, these dependencies have increased the image size by ~.5GB in my testing, but more packages are still needed.

**Comment by calbach** [comment]
Merging 🤞

**Pull request merged**
